lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -4,7 +4,7 @@
 
 <% unless calculator.country_without_consular_facilities? %>
   <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'consular_fees_in_cash_visa_master.govspeak.erb',
+    <%= render partial: 'link_to_consular_fees_for_vietnam.govspeak.erb',
```

Comment:
This change doesn't make sense to me. As far as I can see the content of the template (i.e. the "implementation") is not specific to Vietnam - it's only Vietnam-specific in the sense that it is only *used* in Vietnam-related outcomes. This doesn't seem like a great justification for the new name, although I understand the motivation of wanting a common prefix to match `link_to_consular_fees`.